### ISSUE TITLE
Template 7 - Bug in module positions after playing with modules

### ISSUE THREAD

**Issue by Brightside56** [Joomla3-Admin-template/joomla-cms#33]
I tried to tweak standard modules, but after few change of settings and saving I get only 1 module position in all templates (position-2):

http://sc2.by/pics/images/bugg.png

**Comment by kyleledbetter** [comment]
Yup, thx, the module position select is highly under construction and may be thrown out if we can't get it working in place of the modal window

**Comment by Brightside56** [comment]
huh. then i will do it in mysql xD

**Comment by kyleledbetter** [comment]
I could certainly use the help :) If you'd like to take a stab, this is the file https://github.com/Joomla3-Admin-template/joomla-cms/blob/template7/administrator/components/com_modules/views/module/tmpl/edit_positions.php (assume all my attempted code is wrong :P )
